src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -42,6 +42,12 @@ PowerLevels_e POWERMGNT::FanEnableThreshold = PWR_250mW;
 static int16_t powerValues[] = POWER_OUTPUT_VALUES;
 #endif
 
+static int8_t powerCaliValues[PWR_COUNT] = {0};
```

Comment:
This will need some input from other devs on how to handle factory calibrations.

Reply:
Surely calibration is not needed as the values should be correct from the outset, why would we need extra calibration?

Reply:
Power db is relative with chip, PCB layout and others. There are tolerances for PCB and chip, with mass of products, some tolerances will be very large.

Reply:
add get/set calibration interface at d617ecf

Reply:
@jonesfu I do not understand.. are you saying two identical TX modules could be very different with out put values. As devs we calibrate the TXes and set the values based on our readings. We do not have any power Calibration values. We just set 50mw, 100mw, .. etc based on our power meter readings. You have to play with the values a little but that has worked with ALL of our TXs.

Reply:
@brandonrc In fact, base on our factory test result, the range of tolerance is from -2 db to 1.5 db. For example,  this is 17 dbm value for 50mw, we found the worst db value is 15.23 and highest db value is 19.50.

Reply:
@brandonrc we see our PR is merged into branch menu-oled-joy but power calibration function is removed, we are very concerned the big tolerances of power, hope re-back it. 
Another, The current power calibration code has no effect on other devs If they don't need it.